### PR TITLE
[MIG] product_category_code_unique: Migrate to v15.0 

### PR DESCRIPTION
**Pull request by rolandojduartem**
**Migration to v15.0**

### CODE REVIEW

**Comment by rolandojduartem** [comment]
@luisg123v, This PR is on hold because this module depends on #1036

T#52382

cc @hugho-ad

**Comment by rolandojduartem** [comment]
Hi, @luisg123v, @hugho-ad and @moylop260, please, may you review this pull request? T#52382

cc @desdelinux

**Review by moylop260 — APPROVED** [review]
LGTM

File: product_category_code_unique/__manifest__.py
Comment (luisg123v) [inline product_category_code_unique/__manifest__.py]: Could you add myself as maintainer, please?

File: product_category_code_unique/data/product_category_sequence.xml
Comment (luisg123v) [inline product_category_code_unique/data/product_category_sequence.xml]: name goes first

**Review by luisg123v — CHANGES_REQUESTED** [review]
I think older commits were squashed incorrectly, check bcd7728c64a1af1de97fbe355102d316d83995b0. There an administrative commit was squashed with an autor's one.

In this case, commits shouldn't be squashed, because there's only one administrative commit, so there's nothing to squash with

**Comment by luisg123v** [comment]
CC @rousseldenis

**Comment by rousseldenis** [comment]
@rolandojduartem Thanks for this. Could you follow migrating guidelines especially the 'Full Process' part with pre-commit as you should have two commits (one for black, one for migration)? Thanks

https://github.com/OCA/maintainer-tools/wiki/Migration-to-version-15.0#technical-method-to-migrate-a-module-from-140-to-150-branch

File: product_category_code_unique/__manifest__.py
Comment (rolandojduartem) [reply product_category_code_unique/__manifest__.py]: Done!

File: product_category_code_unique/data/product_category_sequence.xml
Comment (rolandojduartem) [reply product_category_code_unique/data/product_category_sequence.xml]: Done!

**Comment by rolandojduartem** [comment]
Hi, @rousseldenis, as you mentioned, I followed the migrating guideline, but the commit for black does not apply in this case because the code was clean since v14.0, in this PR, there are only changes for migration.

**Comment by rolandojduartem** [comment]
Hi, @luisg123v, changes are done, please, may you review it?

**Review by hugho-ad — APPROVED** [review]

**Review by rousseldenis — APPROVED** [review]

**Review by luisg123v — APPROVED** [review]
LGTM :+1:

**Comment by luisg123v** [comment]
Hi @rousseldenis or @moylop260,

Could you merge, please?

**Pull request merged**
